Assets/MixedRealityToolkit/_Core/Utilities/InputMappingAxisUtility.cs

```diff
@@ -0,0 +1,249 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.Internal.Utilities
+{
+    /// <summary>
+    /// Utility class for Unity's Input Manager Mappings.
+    /// Input from all types should be defined here for use 
+    /// though out the entire toolkit.
+    /// </summary>
+    public static class InputMappingAxisUtility
+    {
+        #region Nested Types
+
+        /// <summary>
+        /// Used to map AxisType from a useful name to the int value the InputManager wants.
+        /// </summary>
+        public enum MappingAxisType
```

Comment:
Shouldn't public types be in their own files?

Reply:
Normally yet, but I want to keep this code isolated, to make it easier to clean up WHEN we can remove it :D

Reply:
Added a remark at the top of the class to state it's purpose.